### PR TITLE
Initial colab notebooks to link in docs

### PR DESCRIPTION
**Pull request by ishanmkh**
- Added colab directory w/ readme
- Initial notebook for use-case: service health monitoring
- Initial notebook for use-case: infra health monitoring
- Initial notebook for use-case: db health monitoring

### CODE REVIEW

File: pxl_scripts/colab_notebooks/README.MD [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Pixie Colab Notebooks
+
+
+## What is Colab?
+
+Colaboratory, or "Colab" for short, is an executable document that lets you write, run and share code within google drive. 
+
+These notebooks are like a [jupyter nodebooks](https://jupyter.org/) stored in google drive. Originally designed for use by the AI/ML community these are great to work with Pixie in a lightweight and interactive manner.
```
Comment (zasgar) [inline pxl_scripts/colab_notebooks/README.MD:8]: nodebooks -> notebook

File: pxl_scripts/colab_notebooks/README.MD [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# Pixie Colab Notebooks
+
+
+## What is Colab?
+
+Colaboratory, or "Colab" for short, is an executable document that lets you write, run and share code within google drive. 
+
+These notebooks are like a [jupyter nodebooks](https://jupyter.org/) stored in google drive. Originally designed for use by the AI/ML community these are great to work with Pixie in a lightweight and interactive manner. 
+
+Few links to learn more about
```
Comment (zasgar) [inline pxl_scripts/colab_notebooks/README.MD:10]: Few links to learn more about Colab:

Comment (ishanmkh) [reply pxl_scripts/colab_notebooks/README.MD]: fixed

File: pxl_scripts/colab_notebooks/README.MD (hunk 1 above)
Comment (ishanmkh) [reply pxl_scripts/colab_notebooks/README.MD]: fixed

**Comment by ishanmkh** [comment]
@zasgar updated the colab to remove all outputs and fixed typos

**Pull request merged**